cypress/e2e/devpool.cy.ts

```diff
@@ -139,7 +140,7 @@ describe("DevPool", () => {
         statusCode: 200,
       });
       // Simulate login token
-      window.localStorage.setItem("sb-wfzpewmlyiozupulbuur-auth-token", JSON.stringify(loginToken));
+      window.localStorage.setItem(`sb-${getSupabaseRef()}-auth-token`, JSON.stringify(loginToken));
```

Comment:
Is it for testing? Can you rename the property name to make this clear?

Reply:
It's just for logging in really I did a find/replace but I guess it's useful for testing too, saves having to replace it in the code then replace it again before pushing